main.go

```diff
@@ -18,7 +18,7 @@ var (
 
 	linodeTokenParamPtr = config.String("linode.token", "", "Required Personal Access Token generated in Linode Console.")
 	regionParamPtr      = config.String("linode.region", "us-west", "Sets the cluster region")
-	hostParamPtr        = config.String("linode.host", hostname(), "Sets the cluster region")
+	linodeLabelParamPtr = config.String("linode.linode-label", linodeLabel(), "Sets the Linode instance label")
```

Comment:
we should probably change these arguments to match.  Instead of all of them been:

```
--linode.token=<...> --linode.region=<...> --linode.node-label=<...>
```
They should probably be:
```
--linode-token=<...> --linode-region=<...> --linode-label=<...>
```

What do you think?

Reply:
Sounds good.  Calling it `linode-label`  will also cause it to match what exists in the docker-machine-linode driver.